winterfell/middlewares/passport.js

```diff
@@ -48,4 +55,25 @@ module.exports = function (app) {
             }
         });
     }
+
+    passport.use('idme', new OAuth2Strategy({
+        authorizationURL: 'https://api.id.me/oauth/authorize',
+        tokenURL: 'https://api.id.me/oauth/token',
+        clientID: idmeClientId,
+        clientSecret: idmeSecretId,
+        callbackURL: '/auth/idme/callback'
```

Comment:
Not 100% sure but I think this needs to be a full url, i.e. https://www.vetafi.org etcc